src/convertToObject.js

```diff
@@ -5,8 +5,106 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const newObject = {};
+
+  const splitStrings = sourceString.split(';');
+
+  const formateStrings = splitStrings.map((param) => {
+    const newString = param
+      .replaceAll('\n', '')
```

Comment:
The `replaceAll` method is not supported in all JavaScript environments. Consider using `replace` with a global regular expression instead: `.replace(/\n/g, '')`.